src/test/java/com/jcabi/github/mock/MkHooksTest.java

```diff
@@ -108,10 +112,14 @@ public void canFetchNonEmptyListOfHooks() throws Exception {
         final Hooks hooks = MkHooksTest.repo().hooks();
         hooks.create(
             // @checkstyle MultipleStringLiterals (1 line)
```

Comment:
@cvrebert does this string literal `"web"` have to be the same for all tests? If so, let's extract it to a `private static final` field and remove the checkstyle suppresions. If not, let's have different string literals for each test.